Sources/plank/Cli.swift

```diff
@@ -144,13 +147,15 @@ func handleGenerateCommand(withArguments arguments: [String]) {
     let includeRuntime: String? = flags[.onlyRuntime] != nil || (flags[.noRuntime] == nil || flags[.noRecursive] != nil) ? .some("") : .none
     let indent: String? = flags[.indent]
     let packageName: String? = flags[.javaPackageName]
+    let nullabilityAnnotationType: String? = flags[.javaNullabilityAnnotationType]
```

Comment:
Can we validate the input here? It seems like I could just enter anything invalid and get `android-support`

Reply:
Done. Moved things into an enum.